pkg/platform/model/buildplanner/project.go

```diff
@@ -57,11 +57,15 @@ func (b *BuildPlanner) CreateProject(params *CreateProjectParams) (strfmt.UUID,
 		}
 	}
 
+	exprB, err := script.MarshalBuildExpression()
```

Comment:
nit: can we make this just `expr`? Seeing `exprB` makes me think there's an `exprA` somewhere and wonder why we need a B.